### ISSUE TITLE
Update usage documentation and use cases

### ISSUE THREAD

**Issue by bigboss2063** [nutsdb/nutsdb#525]
Since the way nutsdb is used has changed, some documents and use cases have not been updated and need to be updated now.

**Comment by TremblingV5** [comment]
I see that there's a repo named `nutsdb-docs` in the repos list of NutsDB. So I think it can be more useful. 

In this repo I found some documents but them seem old. I think we can reuse this repo to publish documents for NutsDB. I'm not quite sure what framework was used in the existing `nutsdb-docs`. Maybe we can perfect it or use some other framework like [https://github.com/facebook/docusaurus](https://github.com/facebook/docusaurus) . `docusaurus` has rich document functions and good supports for multiple versions, multiple languages, and also can publish blogs. 

Totally, I think use a github page to present more rich documents will be better for promoting NutsDB.

**Comment by xujiajun** [comment]
@TremblingV5 Indeed, this project has not been maintained for a long time, resulting in a lot of information being out of date. Currently, it is all markdown files. It seems that the static page is generated using https://jekyllrb.com/. I think your suggestion is very good.
